src/main.rs

```diff
@@ -13,11 +13,25 @@ struct Cli {
 #[derive(Subcommand)]
 enum Commands {
     /// Init a new App Chain config
-    Init,
+    Init {
+        /// App chain name
+        #[arg(short, long = "chain-name")]
+        name: Option<String>,
+        /// Select Sovereign
```

Comment:
let the comment be select DaMode

Reply:
Same as above, it should be exact string